### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by fhaynes**
This adds issue templates so people can more easily provide everything we need to make something actionable.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+## Description
+
+Please note this is not for modifications to existing features. There is another Issue type for that. This is for proposing something that does not yet exist.
+
+## What problem does this solve? What need does it fill?
+A clear and concise description of what the problem/need is. Ex. I'm frustrated when I lose work due to a lack of auto-save.
+
+**Describe the solution you'd like**
```
Comment (LucioFranco) [inline .github/ISSUE_TEMPLATE/new-feature-request.md]: should these be promoted to `##`?

Comment (fhaynes) [reply .github/ISSUE_TEMPLATE/new-feature-request.md]: I dunno. GitHub generated the template.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## What You Expected to Happen
+
+A clear and concise description of what you expected to happen.
+
+## What Actually Happened
+
+A clear and concise description of what actually happened.
+
+## Screenshots
```
Comment (torkleyy) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Having the logging output would also be very helpful.

There already is such a template in `docs/ISSUE_TEMPLATE/bug_report.md`.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Found a bug? Create a report to help us improve!
```
Comment (khionu) [inline .github/ISSUE_TEMPLATE/bug-report.md]: I think "and help us" would be good here, to imply we want them involved in the process beyond just submitting reports.

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+name: New Feature Request
+about: Suggest a new feature for this project
```
Comment (khionu) [inline .github/ISSUE_TEMPLATE/new-feature-request.md]: Tone is a little flat given the rest of the descriptions. How about "Have an idea that could improve this project? We want to hear it!" ?

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (fhaynes) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Ah, I think they have to live under `.github/issues/`. I'll mention the logs in there.

**Comment by fhaynes** [comment]
@khionu @torkleyy do you like this version better?

**Pull request merged**